firebase-firestore/src/testUtil/java/com/google/firebase/firestore/testutil/TestUtil.java

```diff
@@ -418,13 +418,15 @@ public QueryData getQueryDataForTarget(int targetId) {
               }
             });
 
+    SnapshotVersion version = SnapshotVersion.NONE;
+
     for (MaybeDocument doc : docs) {
       DocumentChange change =
           new DocumentChange(updatedInTargets, removedFromTargets, doc.getKey(), doc);
       aggregator.handleDocumentChange(change);
+      version = doc.getVersion().compareTo(version) > 0 ? doc.getVersion() : version;
```

Comment:
Not sure why this worked before but `testQueriesIncludeDocumentsFromOtherQueries()` needs to use `20` as the snapshot version in order to use a higher read time for the newly added doc.